distributions/openhab/src/main/resources/bin/restore

```diff
@@ -12,6 +116,13 @@ setup(){
     exit 0
   fi
 
+  # change file locations to read defaults from based on installed version
+  # according to dpkg only either "openhab" or "openhab2" packages are installed
+  ohPkgName=openhab2
```

Comment:
You won't need to check for this, the master branch is for OH3, the 2.5.X branch is for OH2. So this will really only be "openhab"

Reply:
you sure? I have migrated to OH3 (the "proper" way this time) and the "backup" and "restore" scripts both are located in the OH3 location
/usr/share/openhab(not "2")/runtime/bin/ BUT they still source the env variables from /etc/profile.d/openhab2.sh so backup the OH2 installation not my OH3 one

Reply:
> Are you sure? 

The files that you are editing now get added to a zip file ready for use, at the moment they still contain old paths which is the issue here.

linuxpkg takes that zip and puts the files in the correct locations, it is not aware of the contents of these files and does not change them, so the issue of having outdated references is propagated to the apt install.

The 2.5.x branch in openhab-linuxpkg uses the zip file created by the 2.5.x branch of openhab-distro.
The master branch in openhab-linuxpkg uses the zip file created by the master branch of openhab-distro, this is the up-to-date build for openHAB 3